### DISCUSSION TITLE
Is it possible to simply generate models?

### DISCUSSION THREAD

**Discussion by FishPhd** [MichalLytek/typegraphql-prisma#286] [Q&A] [answered]
One thing that has always bothered me about Prisma is the fact that the schema and any models are decoupled. I'm pretty new to prisma so maybe I am just not understanding the workflow. I came from type-graphql and typeorm where the models and migrations all reference the central entity models you create with typegraphl (where migrations are picked up based on alterations to the model).

I don't like the idea of updating my models and prisma schema separately (I fear this leads to a lack of trust in the models). But I also don't need all the extra generated crud models. 

**So long story short, is it possible to simply use typegraphql-prisma for simply generating the models based on the prisma schema? Allowing me to design resolvers separately**

Im new to the full-stack space so feel free to poke holes or inform me on anything im overlooking.

**Comment by MichalLytek** [comment, marked as answer]
You can just import what you need from the generated folder.

**Reply by FishPhd** [reply]
I guess that's one way. Not the most developer friendly as it dumps a bunch of extra files in the folder one would presumably want to develop the resolvers and edit models. Thanks Michal!

**Reply by MichalLytek** [reply]
https://prisma.typegraphql.com/docs/advanced/emit-blocks

**Reply by FishPhd** [reply]
Wow completely missed this when perusing the docs. Legend!
